pkg/constants/constants.go

```diff
@@ -3,24 +3,37 @@ package constants
 const (
 	SingletonName = "instance"
 	OpenshiftNS   = "openshift-logging"
-	// global proxy / trusted ca bundle consts
-	ProxyName                  = "cluster"
-	TrustedCABundleKey         = "ca-bundle.crt"
-	ClientCertKey              = "tls.crt"
-	ClientPrivateKey           = "tls.key"
+    
+	ProxyName = "cluster"
+
+	TrustedCABundleKey    = "ca-bundle.crt"
+	ClientCertKey         = "tls.crt"
+	ClientPrivateKey      = "tls.key"
 	ClientUsername             = "username"
 	ClientPassword             = "password"
+	GeneratedCAFileName   = "ca.crt"
+	GeneratedKeyFileName  = "system.logging.fluentd.key"
+	GeneratedCertFileName = "system.logging.fluentd.crt"
```

Comment:
@jcantrill: Why do we have fluentd if we are using generic term `collector`. If this is specific to fluentd then we should change variable name to GeneratedKeyFileName->GeneratedFluentdKeyFileName and GeneratedCertFileName->GeneratedFluentDCertFileName. Same questions for FluentdTrustedCAName